core/dcschema.mcf

```diff
@@ -2858,3 +2858,24 @@ Node: dcid:Portion
 typeOf: dcs:UnitOfMeasure
 name: "Portion"
 description: "A unit of measure used to measure serving of food."
+
+Node: dcid:exchangeRate
+typeOf: schema:Property
+name: "exchangeRate"
+description: "The exchange rate of a currency compared to another currency."
+domainIncludes: schema:Currency
+rangeIncludes:UnitPriceSpecification
+
+Node: dcid: currencyIsoCode
+typeOf: schema:Property
+name: "currencyIsoCode"
+description: "The current Iso 4217 code assigned to a currency unit."
+domainIncludes: Currency
```

Comment:
dcs:Currency, schema:Text

Reply:
please don't resolve if comment is not addressed.. (if there was an offline conversation about it, please update the comment here before resolving).

Reply:
I'm pretty sure these were nits of including dcs: tag etc and I so updated them. Is there something that isn't resolved? Current file says:

domainIncludes: dcs:Currency
rangeIncludes: schema:Text

Reply:
gentle ping on this. can you let me know what here is not resolved? thank you!

Reply:
I see, I misread the comment that the issue was in the namespacing. Resolving.